### PR TITLE
Changed test file and modified linux yaml file to resemble the native…

### PR DESCRIPTION
**Pull request by ogibson**
… one

# Background
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

Final step in changing the seurat object to a reduced version of the multisample one. Also modified the linux yaml file that was outdated. 

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ X] Tested locally with Inframock
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ X] Photo of a cute animal attached to this PR

image

### CODE REVIEW

File: docker-compose.linux-dev.yaml [hunk 1]
```diff
@@ -10,25 +10,12 @@ services:
...
       - ./r:/r:cached
       - ./data:/data:cached
-    environment:
-      - EXPERIMENT_ID=e52b39624588791a7889e39c617f669e
     expose:
       - "4000"
     ports:
       - "4000:4000"
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
```
Comment (ivababukova) [inline docker-compose.linux-dev.yaml:20]: why us this deleted?

Comment (ivababukova) [inline docker-compose.linux-dev.yaml:20]: why is this deleted?

Comment (ivababukova) [reply docker-compose.linux-dev.yaml:20]: nevermind, it actually is not, the git diff is confusing

File: docker-compose.linux-dev.yaml [hunk 2]
```diff
@@ -39,7 +26,7 @@ services:
     volumes:
       - ./python:/python:cached
       - ./data:/data:cached
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
     environment:
       - EXPERIMENT_ID=e52b39624588791a7889e39c617f669e
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```
Comment (ivababukova) [inline docker-compose.linux-dev.yaml:32]: why is this added now?

Comment (ogibson) [reply docker-compose.linux-dev.yaml:32]: Oh Iva I didn't get notifications for this or I'd have answered much sooner. This needs to be added to the linux yaml file for the worker to function. I don't exactly know why, but Marcell figured it out and it solved the problems we were having in Linux two months ago

Comment (ivababukova) [reply docker-compose.linux-dev.yaml:32]: I see, cool, I was making sure it is not a copy-pasta out of mistake

**Review by ivababukova — APPROVED** [review]
